### PR TITLE
Compose per integration metadata.yaml and ops_agent_metadata.yaml files based on the content in ops-agent repo.

### PR DESCRIPTION
**Pull request by stackdriver-instrumentation-release**

### CODE REVIEW

**Review by rafaelwestphal — APPROVED** [review]

File: integrations/oracledb/ops_agent_metadata.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 platforms:
 - type: GCE
-  launch_stage: GA
-  install_documentation_url: https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/oracledb
+  launch_stage: HIDDEN
+  launch_date: '2022-11-17'
```
Comment (EvanSimpson) [inline integrations/oracledb/ops_agent_metadata.yaml:4]: @rafaelwestphal This is going to remove the oracle integration from the integration portal. I assume this was unintentional?

Comment (sophieyfang) [reply integrations/oracledb/ops_agent_metadata.yaml:4]: This is due to https://github.com/GoogleCloudPlatform/ops-agent/pull/924 was never merged in.

Comment (EvanSimpson) [reply integrations/oracledb/ops_agent_metadata.yaml:4]: Thanks for looking into that, do we need to re-run the script to generate this PR?

Comment (sophieyfang) [reply integrations/oracledb/ops_agent_metadata.yaml:4]: yes, i just triggered it ->https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/364

Comment (EvanSimpson) [reply integrations/oracledb/ops_agent_metadata.yaml:4]: Thank you!

**Pull request closed without merging**